crypto/crypto.go

```diff
@@ -166,14 +164,12 @@ func HexToECDSA(hexkey string) (*ecdsa.PrivateKey, error) {
 
 // LoadECDSA loads a secp256k1 private key from the given file.
 func LoadECDSA(file string) (*ecdsa.PrivateKey, error) {
-	buf := make([]byte, 64)
-	fd, err := os.Open(file)
+	buf, err := ioutil.ReadFile(file)
```

Comment:
This is super dangerous because if I specify a huge file, it will load it all into memory.

Reply:
Now checking file size with `os.Stats()` before opening it